bears/go/GoMetaLintBear.py

```diff
@@ -0,0 +1,86 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='gometalinter',
+        output_format='regex',
+        output_regex=r'\w+\.go:'
+                     r'(?P<line>\d+):'
+                     r'(?P<column>\d*):'
+                     r'(?P<severity>[a-z]+): '
+                     r'(?P<message>.*) '
+                     r'(?P<additional_info>\([a-z]+\))',
+        severity_map={'error': RESULT_SEVERITY.MAJOR,
+                      'warning': RESULT_SEVERITY.NORMAL})
+class GoMetaLintBear:
+    """
+    Lints your Go files!
+    Concurrently runs a number of Go lint tools.
+    """
+
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(
+        package='github.com/alecthomas/gometalinter', flag='-u')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    SEE_MORE = 'http://gopkg.in/alecthomas/gometalinter.v2'
+
+    def create_arguments(self, filename, file, config_file,
+                         enable: typed_list(str) = (),
+                         enable_all: bool = False,
+                         disable: typed_list(str) = (),
+                         disable_all: bool = False,
+                         gometalinter_config_file: str = '',
+                         ignore_config: bool = False):
+        """
+        :param enable:
+            List of linters to enable.
+            Some linters are disabled by default. Refer goo.gl/gMirRv
+        :param enable_all:
+            Enable all supported linters.
+        :param disable:
+            List of linters to disable.
+        :param disable_all:
+            Disable all supported linters.
+        :param gometalinter_config_file:
+            A JSON configuration file for gometalinter.
+            It overrides ``.gometalinter.json`` which is picked up by
+            default, if present.
+        :param ignore_config:
```

Comment:
what about using `gometalinter_config_file=False` to mean 'ignore config'

Reply:
I'm already using ``gometalinter_config_file``.
Also, when no config file is provided explicitly, ``gometalinter`` automatically takes a file named ``.gometalinter.json`` as a config file. In that case, ``ignore_config`` is used.
I think I should add another config file named ``.gometalinter.json`` to showcase that functionality but it'll then require to turn on ``ignore_config`` in every test.